### ISSUE TITLE
Does not work with `output: 'export'`

### ISSUE THREAD

**Issue by Nakochi-crypto** [dc7290/next-export-optimize-images#608]
**Describe the bug**
Does not work with `output: 'export'` on next.config.js. It was added in Next.js 13.3.

https://nextjs.org/blog/next-13-3#static-export-for-app-router

**To Reproduce**
1. `yarn create next-app`
2. `cd my-app`
3. Edit next.config.js
4. `yarn dev`

next.config.js is like this:

```diff
+const withExportImages = require("next-export-optimize-images")
+
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  output: 'export',
   reactStrictMode: true,
 }
 
-module.exports = nextConfig
+module.exports = withExportImages(nextConfig)
```

**Expected behavior**
It works with `output: 'export'`.

**Screenshots**
N/A

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
N/A

**Additional context**
N/A

**Comment by dc7290** [comment]
Please try to use version 13.3 or higher of Next.js👍

Or try removing `output: "export"` in `next.config.js` and do the build commands `next build && next export && next-export-optimize-images`.

**Comment by Nakochi-crypto** [comment]
I'm using Next.js 13.4.9. Still doesn't work even though `output: 'export'` omitted.

**Comment by dc7290** [comment]
Oops, sorry about that.

What error do you get after you lose the `output`?

**Comment by Nakochi-crypto** [comment]
The issue is fixed in v3.1.0. Thanks!
